### PR TITLE
Minor bugs and features

### PR DESCRIPTION
**Pull request by pierrecagne**
This PR tackles the list of bugs/fetaures from Oddveig:

  - filtering by terrain types (with translation of terrain types in English)
  - some change of mwc to mdc (more to come)
  - new languages for the names of the places (german in particular)
  - maintenance in the edit feature (still some things to clear)

### CODE REVIEW

**Review by cnrdh — APPROVED** [review]
LGTM, except "Generell art" :)

File: src/translate/en.js [hunk 1]
```diff
@@ -48,6 +49,13 @@ export default {
     nb: "Norwegian bokmål",
     nn: "Norwegian nynorsk",
     en: "English",
+    ru: "Russian",
+    fr: "French",
+    is: "Islandic",
```
Comment (filchos) [inline src/translate/en.js]: Could you correct this to Icelandic?

File: src/translate/en.js [hunk 2]
```diff
@@ -68,5 +76,7 @@ export default {
   area: {
     Arktis: "Arctic",
     Antarktis: "Antarctic",
+    ["Generell art"]: "Generic",
```
Comment (filchos) [inline src/translate/en.js:79]: What does "Generell art" mean? Where does it come from?

If needed please make the label English.

**Review by filchos — CHANGES_REQUESTED** [review]
LGTM, just two minor issues.

File: src/translate/en.js (hunk 1 above)
Comment (pierrecagne) [reply src/translate/en.js]: Of course.

File: src/translate/en.js (hunk 2 above)
Comment (pierrecagne) [reply src/translate/en.js:79]: It was a feature asked by Oddveig: sometimes the cases are not located geographically and are "av generell art" (quoting Oddveig on the Norwegian term). The label is not meant to be English, as Artiks and Antarktis (those are the values returned by the API, and they are in Norwegian). The best equivalent in English that came to mind was 'generic' but I'll take anything better!

Comment (cnrdh) [reply src/translate/en.js:79]: About extra (non-)area.

The problem is really:

1. there is a bug changing/removing area from a case.
2. there is no obvious UI to filter on empty area

However, you can already filter on "null". 
First goto: 
https://placenames.npolar.no/?sort=-beginLifespanVersion&area=null

Then press "SAKER" and you should see 84 cases

Ref API
https://v2-api.npolar.no/placename/case/_search?and=area:null&show=name,id,@id
https://v2-api.npolar.no/placename/feature/_search?and=area:null&show=name,id

Comment (pierrecagne) [reply src/translate/en.js:79]: So you recommend I just add the null option in the filtering and tell Oddveig that he can register a case with no area if he wants to be generic?

Comment (cnrdh) [reply src/translate/en.js:79]: Yes, I'd close this and create a new branch for fixing (setting, filtering) cases with no areas.
As well as handling/fixing these. No area is set, but names are connected (26):
https://v2-api.npolar.no/placename/case/_search?and=area:null&not=placenames.id:null&show=name,id,@id,placenames

Comment (cnrdh) [reply src/translate/en.js:79]: The 26 cases with no area AND name(s) are a bit weird, but if a "generic / no area" case is implemented or has consequences for only eg. Svalbard, this is allright, I guess. 

So by handling/fixing I meant really just indicating this situation for the editor, like
"Generic case, but connected to names in: Svalbard (51)":
 
https://stadnamn.npolar.no/sak/283

**Review by cnrdh — APPROVED** [review]

**Comment by pierrecagne** [comment]
I added small changes for case edition. Still one bug in filtering the case search results that I can't understand, I'll open an issue for it

**Pull request merged**